src/style.css

```diff
@@ -0,0 +1,128 @@
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir Book", "Avenir Heavy", Arial, sans-serif;
+}
+
+.bold-text {
+  font-family: "Avenir Heavy", Arial, sans-serif;
+  font-weight: 300;
```

Comment:
also check out with name of font and why do you need to use bold font with font weight 300?